### ISSUE TITLE
Open Package - Identifies back button as "Left hand arrow back button" (Issue 104 from the accessibility list)

### ISSUE THREAD

**Issue by BatJan** [umbraco/Umbraco-CMS#5929]
This is issue 104 on the accessibility list: https://github.com/umbraco/Umbraco-CMS/issues/5277

Screen readers are reading out the ← character, which makes no sense. Therefore is should be wrapped in a span using an aria-hidden="true" attribute.

I'll create a PR shortly.
